homeassistant/components/doorbird.py

```diff
@@ -15,7 +15,7 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.util import slugify
 
-REQUIREMENTS = ['DoorBirdPy==0.1.3']
+REQUIREMENTS = ['doorbirdpy==2.0.4']
```

Comment:
Requirement is now doorbirdpy 2.0.4 but `requirements_all.txt` isn't up to date. Did you run script/gen_requirements_all.py?

Reply:
This has been updated.